a3p-integration/proposals/c:probeZcfBundleCap/package.json

```diff
@@ -0,0 +1,13 @@
+{
+    "agoricProposal": {
+        "type": "/agoric.swingset.CoreEvalProposal",
+        "source": "submission"
```

Comment:
I think the `source` needs to be `subdir`, but I'm not sure

Reply:
It worked just fine this way. Was the "need" you refer a style rule, or is the fact that it works sufficient?

Reply:
`submission` isn't a valid value:
https://github.com/Agoric/agoric-3-proposals/blob/c6e52be97d9c5b77abf72f2ab8896264b398ac47/packages/synthetic-chain/src/cli/proposals.ts#L21-L29

but the logic currenctly doesn't validate params; if it's not `'build'` then it assumes `'subdir'`:
https://github.com/Agoric/agoric-3-proposals/blob/c6e52be97d9c5b77abf72f2ab8896264b398ac47/packages/synthetic-chain/src/cli/build.ts#L36-L37

so please change it to `subdir`